tests/next_tests/__init__.py

```diff
@@ -11,6 +11,77 @@
 # distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
+import enum
+
+import pytest
+
+
+# Skip definitions
+class SkipMark(enum.Enum):
+    XFAIL = pytest.xfail
+    SKIP = pytest.skip
+
```

Comment:
I don't think the enum is needed anymore, since we directly use the pytest functions here
```suggestion
XFAIL = pytest.xfail
SKIP = pytest.skip
```